.travis.yml

```diff
@@ -12,31 +12,28 @@ addons:
     packages:
       - python-pip
 
-services:
-  - docker
-
 before_install:
-  - sudo docker pull docker.io/tripleomaster/centos-binary-qdrouterd:current-tripleo-rdo
+  - sudo echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+  - curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/Release.key | sudo apt-key add -
+  - sudo apt-get update
+  - sudo apt-get upgrade
+  - sudo apt-get install -y podman
 
 install:
   # Install ansible
-  - pip install ansible ansible-lint
+  - pip install ansible ansible-lint molecule[podman]
 
   # Check ansible version
   - ansible --version
 
   # Create ansible.cfg with correct roles_path
-  - printf '[defaults]\nroles_path=../' >ansible.cfg
+  - printf '[defaults]\nroles_path=./' >ansible.cfg
 
 script:
-  # Basic role syntax check
-  - ansible-playbook tests/test.yml -i tests/inventory --syntax-check
+  # Lint test
   - ansible-lint tests/test.yml
-  # Run the role
-  - ansible-playbook tests/test.yml -i tests/inventory --extra-vars "qdr_conf_output_dir=`pwd`/output" && cat `pwd`/output/qdrouterd.conf
-  - sudo docker run -dti --name qdr-test --net=host -v "`pwd`/output/qdrouterd.conf:/etc/qpid-dispatch/qdrouterd.conf" -v "`pwd`/output:/var/log/qdrouterd" -v /var/run:/var/run -v /tmp:/tmp -v "`pwd`/tests/kolla_config_files/:/var/lib/kolla/config_files/" -e KOLLA_CONFIG_STRATEGY=COPY_ALWAYS tripleomaster/centos-binary-qdrouterd:current-tripleo-rdo
-  - sudo docker logs qdr-test
-  - sudo docker exec qdr-test qdstat -b 0.0.0.0:5666 -l
+  # Run the role in a molecule test
+  - molecule test
```

Comment:
In case you were planning on adding additional scenarios, `molecule test` only runs the default scenario, it might be useful to have `molecule test --all`, so that new tests are automatically run